components/ingest-service/backend/elastic/elastic.go

```diff
@@ -42,6 +42,24 @@ func (es *Backend) Initializing() bool {
 	return !es.initialized
 }
 
+func (es *Backend) DeleteProjectTag(ctx context.Context, projectTagToBeDelete string) ([]string, error) {
+	log.Debugf("starting project delete project %q", projectTagToBeDelete)
```

Comment:
Wonder if this message should go later . . . or if we should distinguish initializing the job vs starting it? maybe i'm overthinking

Reply:
Sorry, I don't really know the difference between initializing and starting a job. This log is mostly to show that this function was called.